### ISSUE TITLE
Onboard First Team to TUD

### ISSUE THREAD

**Issue by pjhill** [department-of-veterans-affairs/va.gov-team#35402]
# Description
Onboard a team (add more info here)

**Comment by holdenhinkle** [comment]
Slack DM:
Marci McGuire  9:53 AM
Good morning!  Here is the info I have for test user dashboard:
Keybase Info
Marci McGuire - m_mcguire_va
Adam Whitlock -  adam627
Tom Harrison - tpharrison
Lihan Li - lihanli
Taylor - caffeinateduser
User Table
https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/master/Administrative/vagov-users/mvi-staging-users.csv
Emails for Staging users with particular traits:
https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/master/Administrative/vagov-users/staging-test-accounts-profile.md (edited) 
mvi-staging-users.csv
<https://github.com/department-of-veterans-affairs/va.gov-team-sensitive|department-of-veterans-affairs/va.gov-team-sensitive>department-of-veterans-affairs/va.gov-team-sensitive | Added by GitHub
staging-test-accounts-profile.md
<https://github.com/department-of-veterans-affairs/va.gov-team-sensitive|department-of-veterans-affairs/va.gov-team-sensitive>department-of-veterans-affairs/va.gov-team-sensitive | Added by GitHub

---

Holden's notes:

I created TUD login credentials for Marci and her team and keybased them to everyone.

I reviewed the users in `staging-test-accounts-profile.md` and confirmed they're already in TUD.

---

Slack DM
Marci McGuire  12:57 PM
@Holden Hinkle - Samara advised that her teams use test users listed in these docs:
https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/master/Administrative/vagov-users/staging-test-accounts-myva.md
https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/maste[…]inistrative/vagov-users/staging-test-accounts-direct-deposit.md
https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/maste[…]e/vagov-users/staging-test-accounts-notification-preferences.md
https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/maste[…]strative/vagov-users/staging-test-accounts-myva-payment-info.md

---

Holden's notes:

I created a spreadsheet that listed all the test users noted in the above `md` files and created a Google sheet with the new users which I'll export as `test_users.csv` and to add them to TUD. It took a good few hours to do this, which included manually logging into staging with each user, looking at each user object in a `rails console` session, and making note of each user's `idme_uuid`. We still don't have the `services` information for each user. I pinged John Bramely asking him how we might get that info.

Also, there are two user's that are incomplete:
- vets.gov.user+272@gmail.com (errored 3x on login)
- ruben.moreno@id.me

Sheets:

- https://docs.google.com/spreadsheets/d/19G1_vdSqD8TNeP40eP_v762eZb1dRqY2tUS7IvuSl0A/edit#gid=0
- https://docs.google.com/spreadsheets/d/1bQN3crSLHOxpW6Tx47dF0yGoEj3mKAi-ihCzQVo0CrE/edit#gid=1195873049

**Comment by holdenhinkle** [comment]
John Bramley got back to me about how to get the services associated with each user:

![image.png](https://images.zenhubusercontent.com/5f85bcde4f4764585e8ff820/dbd4ca95-33ed-42a1-b37c-91ec7916bf44)

I found that endpoint in the swagger docs - https://department-of-veterans-affairs.github.io/va-digital-services-platform-docs/api-reference/#/user/getUser

I logged in as each user, got the list of services for each, and added them to the `test_users` sheet. 

I merged a PR to add the new users to `test_users.csv` - https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/pull/388

I then logged into an instance of staging and ran the rake task to add/reset the tud accounts. Now there are 27 tud users.

Joe, Peter and I had a meeting with Marci and 3 engineers and onboarded them to TUD. The meeting went well. One of the engineers had some good feedback:

- search should also search notes
- onKeypress to fire search (not just onClick)
- add 'direct deposit` as a filter
- add search params in url so user's can be shared by sending links

This is ticket is complete.
